src/generic_ui/polymer/cloud-install.ts

```diff
@@ -130,5 +141,9 @@ Polymer({
   },
   ready: function() {
     this.ui = ui;
+    // ID of the latest attempt to create a server, used to distinguish
+    // between install failures that should be flagged to the user and
+    // failures owing to cancellation.
+    this.mostRecentCreateId = 0;
   }
 });
```

Comment:
I thought a `boolean` would suffice initially too but because, if the user cancels install, it currently takes several seconds for the installer to fail - by which time the user may have initiated a whole new install.

I think some of the other changes I'm making, to uproxy-lib, may help eliminate the need for this but for right now I think it has to stay.